tests/psa-client-server/psasim/include/crypto_spe.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright The Mbed TLS Contributors
+ * SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ *
+ */
+
+/**
+ * \file crypto_spe.h
```

Comment:
This file is already present in the repo at `tests/include/spe/crypto_spe.h` - is there no way of using that existing version?

Reply:
In the latest update I removed the `MBEDTLS_PSA_CRYPTO_SPM` (and `crypto_spe.h` as a consequence) because I don't think we need that for our goals here. Wdyt?

Reply:
That's good, yes, thanks